website/docs/plugins/signing.html.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: "registry"
+page_title: "Plugin Signing"
+sidebar_current: "docs-plugins-signing"
+description: |-
+  Terraform plugin signing trust levels
+---
+
+# Plugin Signing
```

Comment:
This document describes plugins in general, but Terraform's tiered signature verification only applies to providers specifically.

Reply:
Yeah, in this case i was hedging a little on the future, as this is a URL hard coded in to a shipped binary, was making it a bit more future proof. I can add some language though that currently only provider plugins are authenticated.